heapster-saw/src/Verifier/SAW/Heapster/Permissions.hs

```diff
@@ -4836,12 +4845,12 @@ remLLVMBLockPermRange rng bp
 remLLVMBLockPermRange rng bp =
   do (bps_l, bp') <-
        if bvInRange (bvRangeOffset rng) (llvmBlockRange bp) then
-         do (bp_l,bp') <- splitLLVMBlockPerm (bvRangeOffset rng) bp
+         do (bp_l,bp') <- splitLLVMBlockPerm (const Nothing) (bvRangeOffset rng) bp
             return ([bp_l],bp')
        else return ([],bp)
      bp_r <-
        if bvInRange (bvRangeEnd rng) (llvmBlockRange bp) then
-         snd <$> splitLLVMBlockPerm (bvRangeEnd rng) bp
+         snd <$> splitLLVMBlockPerm (const Nothing) (bvRangeEnd rng) bp
```

Comment:
Is it worth commenting why these two calls to `splitLLVMBlockPerm` have empty substitutions?

Reply:
Fair enough, I can do that